exercises/22_inc_dec_operators/inc_dec_operators_01.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * This program demonstrates the use of increment (++) and decrement (--) operators in C.
+ * The increment operator adds 1 to its operand, while the decrement operator subtracts 1.
+ * The unusual aspect is that these operators can be used either as prefix or postfix.
+ * 
+ * Prefix (e.g., ++i): Increment/decrement first, then return the new value.
+ * Postfix (e.g., i++): Return the current value first, then increment/decrement.
+ */
+
+// ❌ I AM NOT DONE
+
+#include <assert.h>
+#include <stdbool.h>
+
+int main() {
+	int i = 0;
+	int j = 2;
+	int k = 0;
+	int l = 3;
+
+	// Use ++/-- instead
+	i = i + 1; // <-- Fix here 
+	j = j - 1; // <-- Fix here 
+	
+	// DO NOT CHANGE
+	assert(j == 1);
```

Comment:
did you mean for this to be assert(i == 1)? j is repeated twice here.

Reply:
Yes, thanks!